### DISCUSSION TITLE
Make it easier to clear search filters

### DISCUSSION THREAD

**Discussion by shiruken** [mealie-recipes/mealie#2258] [Feature Request]
**Before submitting this feature request I have**
- [x] Reviewed the documentation and verified that my feature request isn't already a feature.
- [x] Reviewed the existing feature requests and my feature has not been requested.
- [x] Checked the [tasks tagged](https://github.com/hay-kot/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

**Please Describe The Problem To Be Solved**

When browsing by category, tag, or tool via the sidebar, it requires multiple clicks to clear the currently selected options.

![image](https://user-images.githubusercontent.com/867617/225476878-32e35d04-95e7-4a9f-948c-bbe299178105.png)

The user must either click on the group and de-select the currently selected item:

![image](https://user-images.githubusercontent.com/867617/225476917-849769f1-9485-41ee-8ad0-86aa629c603b.png)

Or click the Cog > Reset:

![image](https://user-images.githubusercontent.com/867617/225476998-c9818e9a-6494-4b3e-a265-e326d7cd9e94.png)

**(Optional): Suggest A Solution**

The reset button should be moved into immediate view so the user can easily clear the current selection without requiring multiple clicks. Alternatively, the "X" that appears when searching via text could be used for the same purpose.

![image](https://user-images.githubusercontent.com/867617/225477820-46e5f498-e69d-4ee4-87dd-6cb7bb2bf948.png)

**Additional Information**
- [ ] If the feature is accepted I'm willing to submit a PR to provide this feature

**Comment by michael-genson** [comment]
The homepage button (the Mealie logo in the upper left) clears filters now, similar to the reset button
